app/src/main/java/com/android/streetworkapp/model/parks/ParkLocationRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.android.streetworkapp.model.parks
+
```

Comment:
Suggestion: you could add a general documentation for this interface 
For instance:
```kotlin
/**
 * A repository interface for searching nearby park locations.
 */
```

Reply:
I fixed it. Thank you!